addon/bootstrap.js

```diff
@@ -12,7 +12,7 @@ XPCOMUtils.defineLazyModuleGetter(this, "LegacyExtensionsUtils",
 const prefs = Services.prefs;
 
 // our pref, set it with default false, if it's not there.
-const PREFNAME = "browser.display.truth";
+const PREFNAME = "extensions.pug.lookingglass";
```

Comment:
What's "PUG"? In the soccer world I came from, that's "pick-up-group"; don't think that's it though.

Reply:
![](https://media.giphy.com/media/JfDNFU1qOZna/giphy.gif)

Reply:
Something game related.